vuln/npm/419.json

```diff
@@ -0,0 +1,18 @@
+{
+  "id": 419,
+  "title": "Path Traversal",
+  "overview": "`superstatic` is vulnerable to path traversal on Windows",
+  "created_at": "2018-02-26",
+  "updated_at": "2018-04-29",
+  "publish_date": "2018-04-29",
+  "author": "Сковорода Никита Андреевич (https://github.com/ChALkeR)",
+  "module_name": "superstatic",
+  "cves": [],
+  "vulnerable_versions": "<=5.0.1",
+  "patched_versions": "",
```

Comment:
I _guess_ that `>=5.0.2` or `>5.0.1` could be fine here, as it's fixed in master and there is a `5.0.2` PR which includes that. It's just stalled and not released yet. Thoughts?

Reply:
If you're gonna get some pings on when 5.0.2 would be released maybe you could PR to the repo to update this entry then?